src/test/java/org/broadinstitute/hellbender/utils/test/DataprocTestUtils.java

```diff
@@ -0,0 +1,73 @@
+package org.broadinstitute.hellbender.utils.test;
+
+import org.broadinstitute.hellbender.utils.runtime.ProcessController;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * utilities to help run tests on a Dataproc cluster
+ *
+ * These require you to have a working installation of the google-cloud-sdk (gcloud) and to be authenticated wtih
+ * appropriate permissions
```

Comment:
Document test prerequisites (eg., what environment variables need to be set, etc.)